index.php

```diff
@@ -59,11 +59,18 @@
 
 	$save_path = "{$FROTZ_SAVE_PATH}/{$session_id}.zsav";
 
-	#
+
+
+
+	handle_save_and_restore($command, $save_path, $session_id, $FROTZ_SLOT_SAVES_ENABLED, $FROTZ_SAVE_SLOTS, $FROTZ_SAVE_PATH);
```

Comment:
Not sure why there's so much space around this function, we could probably remove a few.